Gemfile

```diff
@@ -8,3 +8,4 @@ gem "rspec", "~> 3.0"
 gem 'simplecov', require: false, group: :test
 gem 'htmlentities'
 gem 'equivalent-xml'
+gem "unitsml", git: "https://github.com/unitsml/unitsml-ruby.git", branch: "updates/unitsdb_files_path", submodules: true
```

Comment:
Do we need the branch and submodules here?

Reply:
I've merged https://github.com/unitsml/unitsml-ruby/pull/17 now.